index.html

```diff
@@ -0,0 +1,555 @@
+        <title>TNW Clone</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.4.1/semantic.min.css">
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link rel="stylesheet" href="css/main.css">
+        <link rel="stylesheet" type="text/css" media="screen and (max-width: 1025px) and (min-width: 769px)" href="css/tablet.css">
+        <link rel="stylesheet" type="text/css" media="screen and (max-width: 768px)" href="css/mobile.css">  
+    </head>
+    <body>
+        <nav>
+            <div class="top-nav">
+                <ul>
+                    <li><a href="#">News</a></li>
+                    <li><a href="#">TNW2020</a></li>
+                    <li><a href="#">Business <i class="angle down icon"></i></a></li>
+                    <li><a href="#">AMAs</a></li>
+                    <li><a href="#">Spaces</a></li>
+                    <li><a href="#">Terms & Conditions</a></li>
+                </ul>
+                <ul>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Advertise</a></li>
+                    <li><a href="#">Jobs</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
+            </div>  
+        </nav>
+        <main>
+            <div class="fixed-nav">
+                <div class="bottom-nav">
+                    <div class="tnw-logo">
+                        <img alt="TNW-logo" src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg">
+                    </div>
+                    <ul>
+                       <li><a href="#">Latest</a></li>
+                       <li><a href="#">Hard Fork</a></li>
+                       <li><a href="#">Plugged</a></li>
+                       <li><a href="#">Fundamental</a></li>
+                       <li><a href="#">ReadMe</a></li>
+                       <li><a href="#">Growth Quarters</a></li>
+                       <li><a href="#">Neural</a></li>
+                    </ul>
+                    <div class="border-icon">
+                        <div class="icon-border1"><i class="search icon"></i></div>
+                        <div class="icon-border2"><i class="bars icon"></i></div>
+                    </div>
+                      
+               </div>
+               <p class="p1"><a href="#">Did you know we have a TECH FESTIVAL? Check it out <i class="long arrow alternate right icon"></i></a></p>
+           </div>
+            
+            <div class="main-grid">
+                <div class="tnw">
+                    <div>
+                        <img src="./images/tnw-w.png" alt="tnw-logo"/>
+                        <time datetime="2020-03-13 20:00">Friday - March 13, 2020</time>
```

Comment:
- [ ] You date is overlapping the rest of the elements when you change the screen size. Please fix this.